lib/saml_idp_constants.rb

```diff
@@ -5,11 +5,6 @@ module Constants
       LOA1_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/1'.freeze
       LOA3_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/3'.freeze
       REQUESTED_ATTRIBUTES_CLASSREF = 'http://idmanagement.gov/ns/requested_attributes?ReqAttr='.freeze
-
```

Comment:
I was wondering about that too. I recall Sam thinking otherwise. I think he's the one that suggested moving to an initializer. I'll double check.